stubs/extensions/simplexml.phpstub

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/**
+ * @param class-string<SimpleXMLElement> $class_name
+ * @return SimpleXMLElement|false
+ */
+function simplexml_load_file(string $filename, ?string $class_name = SimpleXMLElement::class, int $options = 0, string $namespace_or_prefix = '', bool $is_prefix = false) {}
+
+/**
+ * @param class-string<SimpleXMLElement> $class_name
+ * @return SimpleXMLElement|false
+ */
+function simplexml_load_string(string $data, ?string $class_name = 'SimpleXMLElement', int $options = 0, string $namespace_or_prefix = '', bool $is_prefix = false) {}
+
+/**
+ * @param class-string<SimpleXMLElement> $class_name
+ */
+function simplexml_import_dom(DOMNode $node, ?string $class_name = 'SimpleXMLElement'): ?SimpleXMLElement {}
+
+/**
+ * @param class-string<SimpleXMLElement> $class_name
+ * @php-from 8.1
+ */
+function simplexml_import_dom(SimpleXMLElement|DOMNode $node, ?string $class_name = 'SimpleXMLElement'): ?SimpleXMLElement {}
+
+/**
+ * @implements Traversable<string, SimpleXMLElement>
+ */
+class SimpleXMLElement implements Traversable, Countable
+{
+    /** @return array<array-key, SimpleXMLElement>|null|false */
+    public function xpath(string $expression) {}
+
+    public function registerXPathNamespace(string $prefix, string $namespace): bool {}
+
+    /** @return ($filename is null ? string|false : bool) */
+    public function asXML(?string $filename = null) {}
+
+    /**
+     * @return ($filename is null ? string|false : bool)
+     * @alias SimpleXMLElement::asXML
+     */
+    public function saveXML(?string $filename = null) {}
+
+    public function getNamespaces(bool $recursive = false): array {}
+
+    /** @return array|false */
+    public function getDocNamespaces(bool $recursive = false, bool $fromRoot = true) {}
+
+    public function children(?string $namespaceOrPrefix = null, bool $isPrefix = false): ?SimpleXMLElement {}
+
+    public function attributes(?string $namespaceOrPrefix = null, bool $isPrefix = false): ?SimpleXMLElement {}
+
+    final public function __construct(string $data, int $options = 0, bool $dataIsURL = false, string $namespaceOrPrefix = '', bool $isPrefix = false) {}
+
+    public function addChild(string $qualifiedName, ?string $value = null, ?string $namespace = null): ?SimpleXMLElement {}
+
+    public function addAttribute(string $qualifiedName, string $value, ?string $namespace = null): void {}
+
+    public function getName(): string {}
+
+    public function __toString(): string {}
+
+    public function count(): int {}
+}
+
+/**
+ * @implements RecursiveIterator<string, SimpleXMLElement>
+ */
+class SimpleXMLIterator extends SimpleXMLElement implements RecursiveIterator, Countable
+{
+    public function count(): int {}
+
+    public function rewind(): void {}
+
+    public function valid(): bool {}
+
+    public function current(): SimpleXMLIterator {}
```

Comment:
Technically incorrect, it can return null until PHP 8.1, but I don't think calling `current()` directly is very common, so it's probably preferable to trade the very uncommon false negative for much more common false positives.

Related to php/doc-en#1640